consensus/istanbul/backend/engine.go

```diff
@@ -391,31 +400,28 @@ func (sb *Backend) Prepare(chain consensus.ChainReader, header *types.Header) er
 		if err != nil {
 			return err
 		}
-		bitmap := parentExtra.Bitmap
-		asig := parentExtra.CommittedSeal
+		parentAggregatedSeal := parentExtra.AggregatedSeal
 
-		parentGossipedSeals := sb.core.ParentCommits()
-		if parentGossipedSeals != nil && parentGossipedSeals.Size() != 0 {
-			sb.logger.Trace("backend.Prepare: taking union of parent seals", "valset", parentGossipedSeals.String(), "num", number)
+		additionalParentSeals := sb.core.ParentCommits()
+		if additionalParentSeals != nil && additionalParentSeals.Size() != 0 {
+			logger.Trace("Combining additional seals with the parent aggregated seal", "valset", additionalParentSeals.String(), "num", number, "parentAggregatedSeal", parentAggregatedSeal.String())
```

Comment:
nit:  Should `valset` be renamed to `additionalParentSeals`?

Reply:
IMO `additionalParentSeals` is a more readable name